From 3ab5f70a89f5f6aba3cae240dda55f1d41084397 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Wed, 24 Aug 2022 06:23:08 +0300
Subject: [PATCH] [SW-86869] gaudi3: fix typo in security_load_fw_version fn

This patch fix typo in security_load_fw_version function.

Change-Id: I2fa2e8c6a097027a085b5312bf2ce4a936744e5c
---

diff --git a/arch/gaudi3/arc/security.c b/arch/gaudi3/arc/security.c
index bfe6219..ada7adc 100644
--- a/arch/gaudi3/arc/security.c
+++ b/arch/gaudi3/arc/security.c
@@ -617,7 +617,7 @@
 	uint8_t fw_version_sec = efuse_otp_get_fw_version_security();
 
 	arcboot_rom_status_update(PROGRESS_VER_LOC);
-	/* REVOKE BITS LOCATION: EFUSE */
+	/* FW VERSION LOCATION: EFUSE */
 	arcboot_rom_status_update_location(PROGRESS_VER_LOC_0,
 					   ASIC_EFUSE);
 
